src/features/location-categories/voting-locations/useVotingLocations.tsx

```diff
@@ -2,10 +2,14 @@ import omit from "lodash.omit";
 import { create } from "zustand";
 import { getHashStorage } from "@/utils/zustand";
 import { persist } from "zustand/middleware";
+import cities from "@/data/tr-cities.json";
+
+export type City = (typeof cities)[number];
 
 interface State {
   isOpen: boolean;
   selectedCityId: number | null;
+  selectedCity: City | null;
```

Comment:
let's remove `selectedCityId` then, and only work with `selectedCity`

Reply:
Yes, I had the same idea and I've implemented it on the 3rd commit 🙏